WNPRC_Virology/src/org/labkey/wnprc_virology/WNPRC_VirologyManager.java

```diff
@@ -0,0 +1,16 @@
+package org.labkey.wnprc_virology;
+
+public class WNPRC_VirologyManager
```

Comment:
should the extend the role class?

Reply:
from what I can tell this class doesn't need to extend the role class, but it's there as a placeholder in case there's a use case for it in the future.